samples/bluetooth/broadcast_audio_sink/src/main.c

```diff
@@ -696,6 +856,29 @@ static int init(void)
 		streams[i].stream.ops = &stream_ops;
 	}
 
+	/* Initialize ring buffers and USB */
+#if defined(CONFIG_USB_DEVICE_AUDIO)
+	int ret;
+	const struct device *hs_dev = DEVICE_DT_GET(DT_NODELABEL(hs_0));
+
+	for (size_t i = 0U; i < CONFIG_BT_BAP_BROADCAST_SNK_STREAM_COUNT; i++) {
```

Comment:
```suggestion
	for (int i = 0; i < CONFIG_BT_BAP_BROADCAST_SNK_STREAM_COUNT; i++) {
```

Kconfig values are `int`s